File: altgraph_data_app_java/src/main/java/org/cjoakim/cosmos/altgraph/data/common/model/npm/Maintainer.java
package org.cjoakim.cosmos.altgraph.data.common.model.npm;

import com.azure.spring.data.cosmos.core.mapping.Container;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Instances of this class represent a NPM (Node.js Package Manager) library Maintainer.
 * <p>
 * Chris Joakim, Microsoft, November 2022
 */

@Data
@EqualsAndHashCode(callSuper = false)
@Container(containerName = "npm_graph")
@JsonIgnoreProperties(ignoreUnknown = true)
public class Maintainer extends NpmDocument {

    public Maintainer() {
        super();
    }

    @Override
    public void intern() {

        baseIntern();
    }
}

File: src/altgraph_shared_app/Models/Npm/Maintainer.cs
using Microsoft.Azure.CosmosRepository.Attributes;

namespace altgraph_shared_app.Models.Npm
{
  [Container(Constants.NPM_CONTAINER_NAME)]
  [PartitionKeyPath(Constants.PARTITION_KEY)]
  public class Maintainer : NpmDocument
  {

  }
}